pkgs/tools/package-management/protontricks/default.nix

```diff
@@ -35,6 +36,9 @@ buildPythonApplication rec {
         exec ${lib.getExe steam-run} bash "$@"
       '';
     })
+
+    # Revert vendored vdf since our vdf includes `appinfo.vdf` v29 support
+    ./revert-vendored-vdf.patch
```

Comment:
Can we use `fetchpatch2` with `revert = true`?

Reply:
Oh yep! Was just thinking there should be a better way to do this and just was about to update it before I saw your comment :sweat_smile: